README.md

```diff
@@ -2,34 +2,60 @@
 
 > Interactive colour template previews for the Design System.
 
+Chameleon allows users to change and share different colour palettes using the Australian Government Design System. It is an express server that takes query parameters altering the apperance of the template. 
```

Comment:
Not feeling the express server:

- `express.js` - link to it
More like "Takes query params, modifies a html template and sends back html response"